_posts/2021-11-08-hhvm-4.136.markdown

```diff
@@ -0,0 +1,27 @@
+---
+title: "HHVM 4.136"
+layout: post
+author: atry
+category: blog
+---
+
+HHVM 4.136 is released! HHVM 4.131&ndash;4.135 remain supported, as do the 4.102 and 4.128 LTS releases.
+
+# Highlights
+
+- Add IDE quickfix for missing interface methods.
+- [The `readonly` feature](https://docs.hhvm.com/hack/readonly/introduction) is no longer considered unstable.
+- The `readonly` modifier is now available in the reflection API via the `isReadonly()` or `returnsReadonly()` functions.
+- A new option `-vEval.JitAttemptSuperInlining=true` is added to `hhvm` to enable super inlining optimization.
```

Comment:
- We should document it as the `hhvm.jit_attempt_super_inlining` INI setting
- we've been long-term wanting to move from ini to hdf, which does support that syntax but those should then be provided via an hdf file instead of CLI arguments
- if this is experimental, we should say so, and explain what it is (*if* it's ready for widespread testing)
- if it's not, should just switch the default behavior and remove the option

Reply:
I assume it is experimental because it is in the progress of applying to WWW